wm-system/main/src/main/kotlin/com/ajaxproject/warehouse/service/WaybillServiceImpl.kt

```diff
@@ -2,18 +2,19 @@ package com.ajaxproject.warehouse.service
 
 import com.ajaxproject.api.internal.warehousesvc.output.pubsub.waybill.WaybillCreatedEvent
 import com.ajaxproject.api.internal.warehousesvc.output.pubsub.waybill.WaybillUpdatedEvent
+import com.ajaxproject.warehouse.application.port.ProductRepositoryOutPort
 import com.ajaxproject.warehouse.dto.CustomerDataLiteDto
 import com.ajaxproject.warehouse.dto.WaybillCreateDto
 import com.ajaxproject.warehouse.dto.WaybillDataDto
 import com.ajaxproject.warehouse.dto.WaybillDataLiteDto
 import com.ajaxproject.warehouse.dto.WaybillInfoUpdateDto
 import com.ajaxproject.warehouse.entity.MongoCustomer
-import com.ajaxproject.warehouse.entity.MongoProduct
 import com.ajaxproject.warehouse.entity.MongoWaybill
 import com.ajaxproject.warehouse.exception.InternalEntityNotFoundException
 import com.ajaxproject.warehouse.exception.NotFoundException
+import com.ajaxproject.warehouse.infrastructure.adapter.mongo.entity.MongoProduct
+import com.ajaxproject.warehouse.infrastructure.mapper.mapToMongo
```

Comment:
dependency on infrastructure is not allowed here

Reply:
fixed, check please